config/logstash.yml

```diff
@@ -281,6 +281,9 @@
 #
 # ------------ Other Settings --------------
 #
+# Run Logstash with superuser (default: ALLOW)
+# on_superuser: BLOCK, ALLOW
```

Comment:
I'd recommend we use: run_as_superuser

Reply:
Please take a lok at [this PR](https://github.com/elastic/logstash/pull/14089). Thanks.